src/plugins/dashboard/public/services/embeddable/embeddable.stub.ts

```diff
@@ -16,9 +16,6 @@ export const embeddableServiceFactory: EmbeddableServiceFactory = () => {
   const pluginMock = embeddablePluginMock.createStartContract();
 
   return {
-    getEmbeddableFactory: pluginMock.getEmbeddableFactory,
-    getEmbeddableFactories: pluginMock.getEmbeddableFactories,
-    getStateTransfer: pluginMock.getStateTransfer,
-    EmbeddablePanel: pluginMock.EmbeddablePanel,
+    ...pluginMock,
```

Comment:
I was honestly just being lazy by doing this - but now that I think about it I'm not sure if it makes a difference? 

If a PR on the embeddable side changes something in the plugin mock that we're using, it will fail our tests no matter if we consume the entire mock, or only portions. I think this is ideal, because whoever is changing the mock will have to either fix the failing test, or change the mock so the test continues to work.

I'm also not against listing out everything we are using, especially because that's the pattern elsewhere in our service stubs. What do you think?

Reply:
Hmmm... Will it though? So if something is **removed** from a plugin mock that we use that we are expecting, yes, it will rightfully break - however, if something is **added** to a plugin mock that we do **not** use or expect, I would assume that destructuring the entire the entire mock rather than just selecting the pieces we expect would break things when it probably shouldn't.... Not entirely sure if this is correct, but that was my thought process when being so rigorous about defining the mocks 😆 

Either way, for the sake of consistency with the other stubs, I'm leaning towards being super explicit about which parts of the service we're using 👍

Reply:
I think if something gets added, it'll end up existing for our tests, but it wouldn't break anything because we wouldn't use that service - or expect it not to be present.

That said, from the POV of consistency, I'm convinced and will update this accordingly.